cli-feed-v3.json

```diff
@@ -25,7 +25,7 @@
       "hidden": false
     },
     "v2-prerelease": {
-      "release": "2.36.0",
+      "release": "2.37.0",

```

Comment:
Can you check why 2.36.0 was not moved to v2. I think the rollout is complete and 2.36.0 can be moved to v2. I think @ankitkumarr knows about it.